### ISSUE TITLE
Parse error on empty literal strings.

### ISSUE THREAD

**Issue by virmitio** [coapp/coapp.powershell#8]
The following string will be incorrectly parsed by these tools and will be treated as only opening a string literal:

```
string = @"";  // this comment is still part of the string.
// still going.  ";  // the string is now closed.
```

**Comment by fearthecowboy** [comment]
Fixed in 1.23.207.0 -- To Update, use: 

``` powershell
# Update CoApp PowerShell Tools
Update-CoAppTools -KillPowerShells -Beta 
```

**Comment by virmitio** [comment]
Confirmed fixed in 1.23.207
